### ISSUE TITLE
Add <kbd> styling

### ISSUE THREAD

**Issue by minimalsm** [ethereum/ethereum-org-website#5639]
**Is your feature request related to a problem? Please describe.**

When reviewing #5594, I noticed that we do not have proper styling for `<kbd>` HTML markup. Given we have docs and tutorials that rely on specific user interactions with multiple keypresses, it makes sense to design and implement this.

**Describe the solution you'd like**

We should design our `<kbd>` elements to look like keys.

For example, here is how GitHub has implemented it:

<kbd>CTRL</kbd> + <kbd>C</kbd>

And this is Stackoverflow:

<img width="375" alt="Screenshot 2022-03-10 at 12 15 23" src="https://user-images.githubusercontent.com/62268199/157659784-abf058fd-a8e9-425f-8a82-ca27712879b2.png">

**Additional context**

From [this deploy preview](https://ethereumorgwebsitedev01-chrishobcroftethereumorgwebsit.gtsb.io/en/developers/tutorials/run-light-node-geth/) you can see how it currently looks:

<img width="921" alt="Screenshot 2022-03-10 at 12 16 52" src="https://user-images.githubusercontent.com/62268199/157660105-5953285d-c3ea-4646-8dca-f238b647b7d1.png">

There are many areas of the developer docs and tutorials that we should be using `<kbd>` element, but are not.

**Want to contribute?**

We love contributions from the Ethereum community! Please comment on an issue if you're interested in helping out with a PR.

**Comment by vluna** [comment]
@minimalsm I would be happy to help on this!

**Comment by minimalsm** [comment]
Hey @vluna, happy to have you implement this. Perhaps we should get one of our new designers to whip something up for it first?

**Comment by vluna** [comment]
Sure @minimalsm, I created a couple just for options. You can see [them in this jsfiddle ](https://jsfiddle.net/e702y59d/) and let me know. Once there is an approved design I can look into the update.

**Comment by nloureiro** [comment]
Hello. @vluna I've made a version on [Figma](https://www.figma.com/file/sz9yv6zXVpEx8OgvhtCgFr/%3Ckbd%3E-style?node-id=2%3A13). 

We should change to mono font and reduce the size of the text to 0.9rem, just to give some space to the "key" shaped box not to change the overall line-height when used in the middle of a <p>

**Comment by samajammin** [comment]
Solved in #5651
